public/app/features/explore/QueryRow.tsx

```diff
@@ -201,8 +201,8 @@ export class QueryRow extends PureComponent<QueryRowProps, QueryRowState> {
 
 function mapStateToProps(state: StoreState, { exploreId, index }: QueryRowProps) {
   const explore = state.explore;
-  const item: ExploreItemState = explore[exploreId];
-  const { datasourceInstance, history, queries, range, absoluteRange, queryResponse, latency } = item;
+  const item: ExploreItemState = explore[exploreId]!;
+  const { datasourceInstance, history, queries, range, absoluteRange, queryResponse, latency, eventBridge } = item;
```

Comment:
Just changed to use eventBridge from redux instead of passing it as a prop as we are already connected anyway.